README.md

```diff
@@ -40,6 +41,7 @@ datadog {
     variants {
         fr {
             site = "EU"
+            checkProjectDependencies = "warn"
```

Comment:
Let's not put this in the code snippet, as it will be copy-pasted 90% of the time. The goal of this is to fail at least one if need be.